src/id.rs

```diff
@@ -198,6 +198,9 @@ impl EventId {
     /// neighbors that attempt to communicate with it.
     pub const EVENT_NUM_CLOCKS_OVERFLOWED: EventId =
         EventId(unsafe { NonZeroU32::new_unchecked(EventId::MAX_INTERNAL_ID - 4) });
+    /// Raw nil value used by RaceBuffer. Must not be used as an event id elsewhere
+    pub const EVENT_NIL_VALUE: EventId =
+        EventId(unsafe { NonZeroU32::new_unchecked(EventId::MAX_INTERNAL_ID - 5) });
```

Comment:
If we simply must have a NIL value event, I would like to request it to be at the edge of our internal range (and just bump down the others) rather than having it be at an unexpected offset.

Reply:
In case it wasn't clear, I would rather *not* have such an event, if there's a clean way around it.